python_modules/dagster/dagster/_core/definitions/load_asset_checks_from_modules.py

```diff
@@ -52,7 +52,7 @@ def load_asset_checks_from_modules(
 
 def load_asset_checks_from_current_module(
     asset_key_prefix: Optional[CoercibleToAssetKeyPrefix] = None,
-) -> Sequence[AssetChecksDefinition]:
+) -> Sequence[AssetsDefinition]:
```

Comment:
these should remain the same, no?

Reply:
oops yea need to go back and rechange this.